download_dataset.py

```diff
@@ -0,0 +1,26 @@
+"""Script to download Sentiment140 dataset."""
+
+import argparse
+import io
+import zipfile
+
+import requests
+
+import utils
+
+DATASET_URL = "http://cs.stanford.edu/people/alecmgo/trainingandtestdata.zip"
+
+parser = argparse.ArgumentParser(
+    description="Script to download Sentiment140 dataset. .csv files will be"
+                "placed in {}".format(utils.DATASET_FOLDER))
+
+if __name__ == "__main__":
+    args = parser.parse_args()
+    print("Downloading Sentiment140 dataset...")
+
+    utils.create_dir(utils.DATASET_FOLDER, delete_old=True)
+
+    r = requests.get(DATASET_URL)
+    z = zipfile.ZipFile(io.BytesIO(r.content))
+    z.extractall(utils.DATASET_FOLDER)
```

Comment:
I did not check but if the file is heavy, a progress bar would be a nice touch, especially when using low connection

Reply:
The file is small but this could be a hancement.